### PR TITLE
PHOENIX-5874: IndexTool does not set TTL on its log tables correctly

### PR DESCRIPTION
**Pull request by swaroopak**

### CODE REVIEW

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexVerificationResultRepositoryIT.java [hunk 1]
```diff
@@ -50,24 +67,62 @@ public void testReadResultRow() throws Exception {
...
+            customClock.incrementValue(1000*(DEFAULT_LOG_TTL+5));
+            EnvironmentEdgeManager.injectEdge(customClock);
+            HTable hTable = new HTable(config, RESULT_TABLE_NAME_BYTES);
+            int count = countRows(hTable);
+
+            Assert.assertEquals(0, count);
         }
     }
 
+    public static int countRows(HTable hTable) throws IOException {
```
Comment (gjacoby126) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexVerificationResultRepositoryIT.java:99]: Can be replaced with TestUtil.getRowCount

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexVerificationOutputRepositoryIT.java [hunk 2]
```diff
@@ -39,6 +48,9 @@
 import java.util.List;
 import java.util.Map;
 
+import static org.apache.phoenix.coprocessor.MetaDataProtocol.DEFAULT_LOG_TTL;
+import static org.apache.phoenix.end2end.index.IndexVerificationResultRepositoryIT.countRows;
```
Comment (gjacoby126) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexVerificationOutputRepositoryIT.java:52]: Can use TestUtil.getRowCount

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexVerificationOutputRepositoryIT.java [hunk 3]
```diff
@@ -94,6 +106,33 @@ public void testReadIndexVerificationOutputRow() throws Exception {
...
+            IndexVerificationOutputRepository
+                    outputRepository =
+                    new IndexVerificationOutputRepository(mockStringBytes, conn);
+
+            outputRepository.createOutputTable(conn);
+            TestUtil.assertTableHasTtl(conn, TableName.valueOf(OUTPUT_TABLE_NAME_BYTES), DEFAULT_LOG_TTL);
+            outputRepository.logToIndexToolOutputTable(mockStringBytes, mockStringBytes,
+                    1, 2, mockString, mockStringBytes, mockStringBytes,
+                    EnvironmentEdgeManager.currentTimeMillis(), mockStringBytes, true);
+
```
Comment (gokceni) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexVerificationOutputRepositoryIT.java:126]: Can you assert here that rowCount is greater than 0

Comment (swaroopak) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexVerificationOutputRepositoryIT.java:128]: Done

**Pull request merged**